src/stash/stash.rs

```diff
@@ -81,8 +81,9 @@ pub trait Stash {
     /// and it gets into the known data.
     fn merge(
         &mut self,
-        consignment: Consignment,
-    ) -> Result<Vec<Box<dyn Node>>, Self::Error>;
+        consignment: &Consignment,
```

Comment:
My take was that we should not use a consignment data anymore once we merged them into stash, so better to have that by-value?

Reply:
Oh, I see now: we do nown own consignment data anyway since they are beyond `Arc` after being unmarshalled. Ok, disregard my comment